components/button/button.js

```diff
@@ -7,9 +7,19 @@ const { PropTypes } = React;
 /**
  * General Button component. Use when you need button or a link that looks like button
  */
-function Button({ children, mods = [], size, href, onClick, type, variation, disabled, dataAttrs = {} }) {
+function Button(props) {
+  const {
+    children,
+    size,
+    href,
+    onClick,
+    type,
+    variation,
+    disabled,
+    dataAttrs = {},
+  } = props;
   const restProps = getDataAttributes(dataAttrs);
-
+  const mods = props.mods ? props.mods.slice() : [];
```

Comment:
That was my point ` mods: PropTypes.arrayOf(PropTypes.string),` doesn't express at all how a developer would use it. It doesn't matter though, if we're moving to css-modules soon. And you're right about this is out of scope for this PR.